actix/src/sec-5-arbiter.md

```diff
@@ -1 +1,103 @@
-[WIP] 
\ No newline at end of file
+# Arbiter
+
+Arbiters provide an asynchronous execution context for actors. Where an Actor
+contains a Context that defines it's Actor specific execution state, Arbiters
+host the environment where an actor runs.
+
+As a result Arbiters perform a number of function. Most notably, they are able
+to spawn a new OS thread, run an event loop, execute functions asynchronously
```

Comment:
>execute functions asynchronously

I think it would be better to say `spawn asynchronous tasks` since `execute` may imply blocking and there is no utility for that
As actual tasks are executed by event loop

Reply:
Changed,